<?xml version="1.0" ?>
<launch>

<!--
    <arg name="joy_config" default="ps3" />
    <arg name="joy_dev" default="/dev/input/js4" />

    <node pkg="joy" type="joy_node" name="joy_node">
        <param name="dev" value="$(arg joy_dev)" />
        <param name="deadzone" value="0.05" />
        <param name="coalesce_interval" value="0.1" />
    </node>

    <node pkg="earth_steering" name="joy2cmd_vel" type="joy2cmd_vel" output="screen">
    </node>
    
    <node pkg="earth_steering" name="base2steering" type="base2steering" output="screen">
        <param name="wheelbase" type="double" value="0.44" />
        <param name="delta_max" type="double" value="0.4" />
        <param name="max_x_vel" type="double" value="0.2" />
    </node>
-->

    <node pkg="earth_steering" name="base2steering" type="base2steering" output="screen">
        <param name="wheelbase" type="double" value="0.44" />
        <param name="delta_max" type="double" value="0.4" />
        <param name="max_x_vel" type="double" value="1.0" />
    </node>

</launch>
